File: insumo.cs
public class insumo{
	private int _idInsumo;
	private string _descripcion;
	private unidadMedida _unidad;

	public  insumo(){}

	public insumo(int _idInsumo, string _descripcion, unidadMedida _unidad){
		this._idInsumo = _idInsumo;
		this._descripcion = _descripcion;
		this._unidad = _unidad;
	}

	public int IdInsumo{
		get{
			return _idInsumo;
		}
		set{
			_idInsumo = value;
		}
	}

	public string Descripcion{
		get{
			return _descripcion;
		}
		set{
			_descripcion = value;
		}
	}

	public unidadMedida Unidad{
		get{
			return _unidad;
		}
		set{
			_unidad = value;
		}
	}
}

File: java/Insumo.java
enum unidadMed{kg,lt,cajas,unid};
public class Insumo{
	private int idInsumo;
	private String descripcion;
	private unidadMed unidMed;
	
	public Almacen(int idInsumo,String descripcion,unidMed unidadMed){
		this.idInsumo=idInsumo;
		this.descripcion=descripcion;
		this.unidadMed=unidadMed;
	}
	
	public void setidInsumo(int idInsumo){
		this.idInsumo=idInsumo;
	}
	public int getidInsumo(){
		return this.idInsumo;
	}
	
	public void setdescripcionm(String descripcion){
		this.descripcion=descripcion;
	}
	public String getdescripcion(){
		return this.descripcion;
	}
	
	public void setunidMed(unidadMed unidMed){
		this.unidMed=unidMed;
	}
	public unidadMed getunidMed(){
		return this.unidMed;
	}
	
}